src/dists.ad.js

```diff
@@ -1322,6 +1322,10 @@ var Marginal = makeDistributionType({
     this.supp = _.map(this.params.dist, function(obj) {
       return obj.val;
     });
+
+    this.getDist = function() {
```

Comment:
Sounds good. I think `hist` isn't clearly enough of an improvement that it's worth switching over now, but it's worth keeping in mind when we write future code that involves these concepts.